README.md

```diff
@@ -5,15 +5,16 @@
 [![CircleCI](https://circleci.com/gh/GPflow/GPflow/tree/develop.svg?style=svg)](https://circleci.com/gh/GPflow/GPflow/tree/develop)
 [![Coverage Status](http://codecov.io/github/GPflow/GPflow/coverage.svg?branch=master)](http://codecov.io/github/GPflow/GPflow?branch=master)
 [![Documentation Status](https://readthedocs.org/projects/gpflow/badge/?version=master)](http://gpflow.readthedocs.io/en/master/?badge=master)
+[![Slack Status](https://img.shields.io/badge/slack-gpflow-green.svg?logo=Slack)](https://join.slack.com/t/gpflow/shared_invite/enQtOTE5MDA0Nzg5NjA2LTYwZWI3MzhjYjNlZWI1MWExYzZjMGNhOWIwZWMzMGY0YjVkYzAyYjQ4NjgzNDUyZTgyNzcwYjAyY2QzMWRmYjE)
+
 
 [Website](https://gpflow.org) |
 [Documentation (develop/v2.0)](https://gpflow.readthedocs.io/en/develop/) |
 [Documentation (v1.5)](https://gpflow.readthedocs.io/en/v1.5.1-docs/) |
 [Glossary](GLOSSARY.md)
 
-GPflow is a package for building Gaussian process models in python, using [TensorFlow](http://www.tensorflow.org). It was originally created and is now managed by [James Hensman](http://jameshensman.github.io/) and [Alexander G. de G. Matthews](http://mlg.eng.cam.ac.uk/?portfolio=alex-matthews).
-The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contributors) (in alphabetical order) is
-Alessandro Davide Ialongo, Alexander G. de G. Matthews, Alexis Boukouvalas, [Artem Artemev](http://github.com/awav/), Daniel Marthaler, David J. Harris, Eric Hambro, Hugh Salimbeni, Ivo Couckuyt, James Hensman, Keisuke Fujii, Mark van der Wilk, Mikhail Beck, Pablo Leon-Villagra, Rasmus Bonnevie, Sergio Pascual-Diaz, ST John, Tom Nickson, Valentine Svensson, Vincent Dutordoir, Zoubin Ghahramani.
+GPflow is a package for building Gaussian process models in Python, using [TensorFlow](http://www.tensorflow.org). It was originally created by [James Hensman](http://jameshensman.github.io/) and [Alexander G. de G. Matthews](https://github.com/alexggmatthews). It is now actively maintained by (in alphabetical order) [Alexis Boukouvalas](https://github.com/alexisboukouvalas), [Artem Artemev](http://github.com/awav/), [Eric Hambro](erichambro.com), [Joel Berekely](https://github.com/joelberkeley), [James Hensman](http://jameshensman.github.io/), [Mark van der Wilk](https://markvdw.github.io/), [ST John](https://github.com/st--) and [Vincent Dutordoir](https://vdutor.github.io/).
```

Comment:
biggest change is that I moved the list contributors out and only highlight the active maintainers. Are you guys happy with this?

Reply:
related question - what do we do with copyright notices and mentioning authors of notebooks ?

Reply:
https://github.com/joelberkeley -> https://github.com/joelberkeley-pio

Reply:
Berekely -> Berkeley

Reply:
@joelberkeley-pio Sorry for misspelling your name. I'll update it.

Reply:
@st-- I would leave that stuff as is (for now).

Reply:
Maybe add a link to the github contributors page in place of the deleted names?